withings-lib/withings-lib.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for withings-lib
+// Project: https://www.npmjs.com/package/withings-lib
+// Definitions by: mpicciolli <https://github.com/mpicciolli>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var WithingsClient:WithingsClient;
+
+declare module 'withings-lib' {
+    export = WithingsClient;
+}
+
+interface WithingsClient {
```

Comment:
http://definitelytyped.org/guides/best-practices.html#ghost-modules
please check this document.